src/detail-page/DetailPage.tsx

```diff
@@ -40,6 +41,7 @@ export const DetailPage: React.FC<{}> = () => {
               profession={profession}
             />
           </TabContainer>
+          <DemoChart />
```

Comment:
Maybe we can add some comment here and before import that this is temporary? :)

Reply:
I'd even go a step further and remove the use of component (line 44) before merge to `master`. After all, we have a live production site, so we should take care of what we release 😎

Reply:
Done